packages/rum/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -132,12 +135,18 @@ function newView(
 
   const { stop: stopActivityLoadingTimeTracking } = trackActivityLoadingTime(lifeCycle, setActivityLoadingTime)
 
+  const { stop: stopCLSTracking } = trackLayoutShift(lifeCycle, (layoutShift) => {
+    cumulativeLayoutShift = (cumulativeLayoutShift || 0) + layoutShift
+    scheduleViewUpdate()
+  })
```

Comment:
```suggestion
  if (isLayoutShiftSupported()) {
    cumulativeLayoutShift = 0 
    const { stop: stopCLSTracking } = trackLayoutShift(lifeCycle, (layoutShift) => {
      cumulativeLayoutShift = cumulativeLayoutShift + layoutShift
      scheduleViewUpdate()
    })
  }
```
then we would avoid to handle undefined case in callback.
It could probably remove the need to check the support in the track function.

Reply:
We still need to handle the `undefined` case, but it's a bit easier to follow, so I just used a `!` operator. 👍